symtabAPI/src/emitElfStatic-ppc64.C

```diff
@@ -1091,7 +1099,7 @@ bool emitElfStatic::handleInterModuleSpecialCase(Symtab * /*target*/,
   // or create it. 
   //
 
-  unsigned int *td = (unsigned int *)(data + dest - (dest % 4));
```

Comment:
The `(dest % 4)` seems sketchy; I would guess that `(dest % 4)` is always 0, otherwise this is not reading the instruction where `dest` points, but 1-3 bytes before.  It does force it to be 4-byte aligned.